src/ProjectTemplates/test/EmptyWebTemplateTest.cs

```diff
@@ -32,7 +32,7 @@ public ITestOutputHelper Output
         }
 
         [ConditionalFact]
-        [SkipOnHelix("Cert failures", Queues = "All.OSX")]
+        [SkipOnHelix("Cert failure, https://github.com/dotnet/aspnetcore/issues/28090", Queues = "All.OSX;Windows.10.Arm64v8.Open;(Debian.9.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036")]
```

Comment:
nit: `Windows.10.Arm64v8.Open;(Debian.9.Arm64.Open)Ubuntu.1804.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-9-helix-arm64v8-a12566d-20190807161036` could be a const. Avoid churn when it changes

Reply:
Where would you recommend putting it? On the attribute itself?

Reply:
I was thinking a static constants class in ProjectTemplates test project so it can be shared in this PR.

Maybe a static constants class in the project that contains SkipOnHelixAttribute? That would allow it to be shared everywhere.